resources/views/matriculate/create.blade.php

```diff
@@ -0,0 +1,105 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-12">
+            @role('admin')
```

Comment:
The role `admin` should be granted just for features that are exclusive to them. Since this can be accessed by a `treinador`, we should use a `can('treinador')`.